data/coverage/2024.yml

```diff
@@ -87,3 +87,12 @@ july:
     - type: video
       title: "LRUG July 2024 - Joel Biffin - Finding unused Ruby methods"
       url: https://assets.lrug.org/videos/2024/july/joel-biffin-finding-unused-ruby-methods-lrug-jul-2024.mp4
+september:
+  from-spaghetti-to-lasagna-layering-your-code-with-ddd:
+    - type: video
+      title: "LRUG September 2024 - Lily Stoney - From Spaghetti to Lasagna/ Layering your code with DDD"
```

Comment:
[nitpick]

There is a `/` after `Lasagna`.

Reply:
Oh, good catch. I'm not sure how that ended up there